tests/modules/multiauth/lib/Auth/Source/MultiAuthTest.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+namespace SimpleSAML\Test\Module\multiauth\Auth\Source;
+
+use PHPUnit\Framework\TestCase;
+use SimpleSAML\Configuration;
+use SimpleSAML\Module\multiauth\Auth\Source\MultiAuth;
+
+class MultiAuthTest extends TestCase
```

Comment:
Since the test is manipulating the configuration, it should probably extend `\SimpleSAML\Test\Utils\ClearStateTestCase` in order to avoid state spill over tests.